packages/orderbook/src/lib/orderbook/orderbook.spec.ts

```
@@ -1,123 +1,183 @@
+  MatchedOrder,
+} from './orderbook.types';
+// import { Asset, Chains } from '../asset';
+
+describe('orders provider', async () => {
+  const orderbookApi = 'https://orderbook-stage.hashira.io';
+  const address = '0x52FE8afbbB800a33edcbDB1ea87be2547EB30000';
+  const id = 'c519b735bcef0a6bd6a54ac7d46449087b2146cae75541c2ddde686cf8fba294';
+
+  const orderbook = new Orderbook(new Url(orderbookApi));
+
+  test.only('should get order', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)